plugins/chat/assets/javascripts/discourse/components/chat/thread/list-item-unread-indicator.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if this.showUnreadIndicator}}
```

Comment:
Not sure about this component because it's almost identical to `header-unread-indicator` but they both have slightly different logic...

Reply:
Duplication is fine to me for now